cmd/crowdsec/output.go

```diff
@@ -125,8 +125,6 @@ LOOP:
 			if len(cache) > 0 {
 				cacheMutex.Lock()
 				cachecopy := cache
-				newcache := make([]types.RuntimeAlert, 0)
-				cache = newcache
```

Comment:
suppressing this is going to skip "emptying" the cache no ?

Reply:
If I'm not mistaken, we are exiting the function after this block (break LOOP), and cache disappears